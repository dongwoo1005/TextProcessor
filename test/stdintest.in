stdin doublewords
hi how are you
stdin Hello World! dropfirst 1
stdin dropfirst 1 Hello World!
stdin Hello dropfirst 1 World!
stdin Hello dropfirst World! 1
