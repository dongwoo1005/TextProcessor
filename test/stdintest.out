hi
hi
how
how
are
are
you
you
stdin
stdin
Hello
Hello
World!
World!
dropfirst
dropfirst
1
1
stdin
stdin
dropfirst
dropfirst
1
1
Hello
Hello
World!
World!
stdin
stdin
Hello
Hello
dropfirst
dropfirst
1
1
World!
World!
stdin
stdin
Hello
Hello
dropfirst
dropfirst
World!
World!
1
1
